pkg/virt-handler/heartbeat/heartbeat_test.go

```diff
@@ -36,6 +36,25 @@ var _ = Describe("Heartbeat", func() {
 		}
 		fakeClient = fake.NewSimpleClientset(node)
 	})
+	Context("upon finishing", func() {
+		It("should set the node to not schedulable", func() {
+			heartbeat := NewHeartBeat(fakeClient.CoreV1(), deviceController(true), config(), "mynode")
+			stopChan := make(chan struct{})
+			done := heartbeat.Run(30*time.Second, stopChan)
```

Comment:
So we're setting a heartbeat interval of 30 seconds? Does this mean the test can take 30 seconds to run?

Reply:
https://github.com/kubevirt/kubevirt/blob/9b85bf7799b2f7b01c7ae5ac9a41537e2f5b4f45/pkg/virt-handler/heartbeat/heartbeat.go#L73
You're welcome :)